src/main/java/org/assertj/core/error/ShouldBeEqual.java

```diff
@@ -140,7 +148,10 @@ private boolean actualAndExpectedHaveSameStringRepresentation() {
    *          of object
    * @return the error message from description using {@link #expected} and {@link #actual} basic representation.
    */
-  private String defaultErrorMessage(Description description, Representation representation) {
+  protected String defaultErrorMessage(Description description, Representation representation) {
```

Comment:
why has this changed to `protected`  ?

Reply:
Because I was trying something else and have forgotten to revert those changes. I'll fix it.